csm/core/controllers/routes.py

```diff
@@ -31,6 +31,7 @@
 from csm.core.controllers.rgw.s3.bucket import S3BucketView
 from csm.core.controllers.information import VersionInformationView
 from csm.core.controllers.activities import ActivitiesListView, ActivitiesView
+from csm.core.controllers.information import TopologyView, ResourcesTopologyView, ResourceTopologyView, AllViews, SpecificView
```

Comment:
The class names does not look good. Please change it so that it signifies with our requirements

Reply:
Updated proper class name